### PR TITLE
Plamut sd 2870

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

**Comment by mugurrus** [comment]
:+1:

**Comment by sivakuna-aap** [comment]
:+1:

File: client/spec/authoring_spec.js [hunk 1]
```diff
@@ -178,4 +179,12 @@ describe('authoring', function() {
         expect(authoring.getHistoryItems().count()).toBe(1);
         expect(authoring.getHistoryItem(0).getText()).toMatch(/Published by.*/);
     });
+
+    it('allows to create a new empty package', function () {
+        monitoring.openMonitoring();
+        browser.sleep(500);
```
Comment (plamut) [inline client/spec/authoring_spec.js]: (while we are waiting for the build to start...)

I have noticed this practice of using `browser.sleep()` in several tests, but I'm not sure this is the right approach. In general, shouldn't we use protractor's promises returned by triggered actions (`click()`, `setLocation()`, etc.) instead to wait for these actions to complete? Or am I missing something?

Thanks.

**Comment by plamut** [comment]
Seems like the build is passing. I guess navigating to /monitoring instead of /authoring did the trick? I'll try in the other PR, too.

**Comment by plamut** [comment]
A workaround through the monitoring view did the trick, for some reason it was not possible to navigate to `/authoring`. Thanks @ioanpocol

**Pull request merged**